fuzz/fuzz_targets/serializer_cmp.rs

```diff
@@ -93,7 +93,8 @@ pub fn compare_back_references(allocator: &Allocator, node: NodePtr) -> io::Resu
         }
         match result1 {
             Some(path) => {
-                f.write_all(&[BACK_REFERENCE])?;
+                f.write_all(&[BACK_REFERENCE])
+                    .map_err(|_| clvmr::error::EvalErr::SerializationError)?;
```

Comment:
instead of mapping this explicitly, you can make `EvalErr` implement `From<std::io::Error>` for this conversion

Reply:
If you use `#[from] io::Error` with `thiserror`, it will do that for you

Reply:
i solved this a different way because i wanted the error to not require an io error.